iothub/device/src/Transport/Http/HttpTransportHandler.cs

```diff
@@ -70,7 +70,7 @@ internal sealed class HttpTransportHandler : TransportHandler
             ProductInfo productInfo = context.ClientConfiguration.ClientOptions.ProductInfo;
             _deviceId = context.ClientConfiguration.DeviceId;
             _moduleId = context.ClientConfiguration.ModuleId;
-            Uri httpsEndpoint = new UriBuilder(Uri.UriSchemeHttps, context.ClientConfiguration.GatewayHostName).Uri;
+            Uri httpsEndpoint = new UriBuilder(Uri.UriSchemeHttps, context.ClientConfiguration.IotHubHostName).Uri;
```

Comment:
@abhipsaMisra It couldn't find GatewayHostName so I changed it to this. Is this right?

Reply:
Never mind.